File: java/com/epicdragonworld/gameserver/network/packets/sendable/ChatResult.java
package com.epicdragonworld.gameserver.network.packets.sendable;

import com.epicdragonworld.gameserver.network.SendablePacket;

/**
 * @author Pantelis Andrianakis
 */
public class ChatResult extends SendablePacket
{
	public ChatResult(byte chatType, String sender, String message)
	{
		// Send the data.
		writeShort(10); // Packet id.
		writeByte(chatType); // 0 system, 1 normal chat, 2 personal message
		writeString(sender);
		writeString(message);
	}
}

File: EpicDragonWorldServer/Network/Packets/Sendable/ChatResult.cs
/**
 * Author: Pantelis Andrianakis
 * Date: November 7th 2018
 */
public class ChatResult : SendablePacket
{
    public ChatResult(byte chatType, string sender, string message)
    {
        // Send the data.
        WriteShort(11); // Packet id.
        WriteByte(chatType); // 0 system, 1 normal chat, 2 personal message
        WriteString(sender);
        WriteString(message);
    }
}
